helper_test.go

```diff
@@ -55,13 +72,49 @@ func TestMGetCache(t *testing.T) {
 			DoFn: func(cmd Completed) RedisResult {
 				return slotsResp
 			},
+			DoMultiFn: func(cmd ...Completed) *redisresults {
+				return &redisresults{s: []RedisResult{newErrResult(context.Canceled), newErrResult(context.Canceled)}}
+			},
 		}
 		client, err := newClusterClient(&ClientOption{InitAddress: []string{":0"}}, func(dst string, opt *ClientOption) conn {
 			return m
 		})
 		if err != nil {
 			t.Fatalf("unexpected err %v", err)
 		}
+		disabledCacheClient, err := newClusterClient(&ClientOption{InitAddress: []string{":0"}, DisableCache: true}, func(dst string, opt *ClientOption) conn {
```

Comment:
I have some doubts about this, and it is timing out, Can you take a look? @rueian

Reply:
Hi @SoulPancake, when the `DisableCache` is set, you should mock the m.DoMultiFn instead of the m.DoMultiCacheFn.

Reply:
Done, Can you please review @rueian